src/components/menus/wallet_settings.vue

```diff
@@ -647,6 +727,32 @@ export default {
       this.modals.change_password.new_password = "";
       this.modals.change_password.new_password_confirm = "";
     },
+    async doExportTransfers() {
+      this.hideModal("export_transfers");
+
+      let passwordDialog = await this.showPasswordConfirmation({
+        title: this.$t("dialog.exportTransfers.title"),
+        noPasswordMessage: this.$t("dialog.exportTransfers.message"),
+        ok: {
+          label: "export".toLocaleUpperCase(this.locale),
+          color: "primary"
+        },
+        dark: this.theme == "dark",
```

Comment:
<img width="438" alt="Screen Shot 2021-04-27 at 9 18 36 am" src="https://user-images.githubusercontent.com/8342557/116162806-6049cd80-a6ab-11eb-957c-a7a4cb4d92de.png">

This should be white, with the same blue button... ik a few of the other ones I missed are still dark, but yeah, should be white

Reply:
Confirming both buttons should be white if dark mode is selected?

Reply:
The button colours can stay the same tbh, just the background to be dark. "show private keys" is an example of what I mean.